lib/config.js

```diff
@@ -5,4 +5,9 @@
 
 const {config} = require('bedrock');
 
-config.profile = {};
+config.profile = {
+  zcap: {
```

Comment:
My only concern here is that this isn't very specific. These are `zcap` config vars for "what"? This appears to be some zcap config parameters for the zcaps that the profile delegates. It's fine not to be specific if config vars are going to apply generally, but if we need different config vars for other cases we'll have a problem. I guess I'll go with the YAGNI principle here and accept it as-is, but wanted to make this note so we can make sure we're thinking about this sort of thing when we name things.

In other words, our naming process should not involve *just* considering the current use case/thing we're making a PR for, but we should be considering usage more generally -- and what others (e.g., maintainers/users) will think when they come across this in the future ... we want them to have an immediate guess as to what this is for and we want their guess to be correct.